bedrock/firefox/templates/firefox/new/base.html

```diff
@@ -8,7 +8,7 @@
 
 {% block page_title_prefix %}{{_('Download Firefox')}} — {% endblock %}
 {% block page_title %}{{_('Free Web Browser')}}{% endblock %}
-{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, Mac OS X, Linux, Android and iOS today!')}}{% endblock %}
+{% block page_desc %}{{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!')}}{% endblock %}
```

Comment:
I guess one thing we also need to be mindful of here is also SEO impact. I'd like to run this change by Peter to get his thoughts on potential impact (if any), and how to best mitigate while still updating our terminology.